pages/reference-smart-account/owners/swapOwner.mdx

```diff
@@ -0,0 +1,88 @@
+import { Tabs, Callout } from 'nextra/components'
+
+# `swapOwner`
+
+Replaces the owner `oldOwner` in the Safe with `newOwner`.
+
+<Callout type='warning'>This can only be done via a Safe transaction.</Callout>
+
+## Usage
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['example.sol']}>
+  <Tabs.Tab>
+    ```solidity
+    interface ISafe {
+        function swapOwner(
+            address prevOwner,
+            address oldOwner,
+            address newOwner
+        ) external
+    }
+
+    contract Example {
+        function example() … {
+            (ISafe safe).swapOwner(
+                "0x...",
+                "0x...",
+                "0x..."
+            )
+        }
+    }
+    ```
+
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Parameters
+
+### `prevOwner`
+
+- **Type:** `address`
+
+Owner that pointed to the owner to be replaced in the linked list
+
+```solidity focus=2
+swapOwner(
+    "0x...",
+    "0x...",
+    "0x..."
+)
+```
+
+### `oldOwner`
+
+- **Type:** `address`
+
+Owner address to be replaced.
+
+```solidity focus=3
+swapOwner(
+    "0x...",
+    "0x...",
+    "0x..."
+)
+```
+
+### `newOwner`
+
+- **Type:** `address`
+
+New owner address.
+
+```solidity focus=4
+swapOwner(
+    "0x...",
+    "0x...",
+    "0x..."
+)
+```
+
+## Events
+
+### [`AddedOwner`](./addOwnerWithThreshold.mdx#addedowner)
+
+### [`RemovedOwner`](./removeOwner.mdx#removedowner)
```

Comment:
Events should be well documented everywhere instead of linking to other methods, even if it's the same content.